src/main/java/io/bobba/poc/core/rooms/RoomManager.java

```diff
@@ -62,14 +62,48 @@ private void loadModelsFromDb() throws SQLException {
         }
 	}
 	
-	private void createDummyRoom() {
-		RoomData roomData = new RoomData(roomId++, "The deep forest", "Relevance", "a very cool room", 25, "", "model_h", LockType.Open);
-		Room room = new Room(roomData, getModel(roomData.getModelId()));		
-		this.rooms.put(room.getRoomData().getId(), room);
-		
-		roomData = new RoomData(roomId++, "dot dot dot", "Gravity", "a cool room", 25, "", "model_g", LockType.Open);
-		room = new Room(roomData, getModel(roomData.getModelId()));
-		this.rooms.put(room.getRoomData().getId(), room);
+	private void loadRoomsFromDb() throws SQLException {
+
+    System.out.println("Carregando salas...");
+		int loadedRooms = 0;
+    try (Connection connection = BobbaEnvironment.getGame().getDatabase().getDataSource().getConnection(); Statement statement = connection.createStatement()) {
```

Comment:
This approach loads all the room at the start. Each room should be loaded dynamically when someone wants to enter on it... So this fix shouldn't be focused on the database but in the "Room Management" thing :(